### PR TITLE
Feature/landing page

### PR DESCRIPTION
**Pull request by PaigeVannelli**
## Is this a fix or a feature?
Feature 

## What is the change?
Added Router, BrowserRouter, and Switch to our main app to route from the main page to the register page. Add Link to LandingPage component to route users to register

## What does it fix?
n/a

## Where should the reviewer start?
Review the App and LandingPage components 

## How should this be tested?
open app to view LandingPage and click through to register

### CODE REVIEW

File: src/components/App/App.tsx [hunk 1]
```diff
@@ -16,43 +18,60 @@ const App = () => {
...
+                    setPage={setPage}
+                    inputs={[{property: destination, method: setDestination, placeholder: 'Destination'}]} 
+                  />
+                }
+                </>
+              )
+            }}
+          />
+        </Switch>
+      </BrowserRouter>
```
Comment (alexthompson207) [inline src/components/App/App.tsx:74]: Great job setting up router!

File: src/components/LandingPage/LandingPage.tsx [hunk 2]
```diff
@@ -1,8 +1,13 @@
...
   return (
-    <div>
-
+    <div className='signup'>
+      {/* <img>LOGO</img> */}
+      <Link to="/register">
+        <button className='signup__button'>Sign up now</button> 
+      </Link>
     </div>
   )
```
Comment (alexthompson207) [inline src/components/LandingPage/LandingPage.tsx:13]: This looks great!

**Review by alexthompson207 — APPROVED** [review]
Awesome job with this Paige! I think next steps for this view would be finding a good logo!

File: src/components/App/App.tsx (hunk 1 above)
Comment (itsnameissteven) [inline src/components/App/App.tsx:74]: Thanks for setting up the router this is great. Im about ready to merge in my branch so a couple things might change.

**Review by itsnameissteven — APPROVED** [review]
I see no issues here. I think this is ready to merge.

**Pull request merged**